docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -566,6 +566,13 @@ Write operations are supported for tables stored on the following systems:
   detects write collisions on ADLS Gen2 when writing from multiple Trino
   clusters, or from as any number of open source Delta Lake clusters.
 
+* Google Cloud Storage
```

Comment:
If this is treated the same as Azure ADLS, we can probably combine them into one bullet point, e.g. `* Azure ADLS Gen2, Google Cloud Storage`

Then "Writes to the Azure ALDS Gen2 and Google Cloud Storage file systems are enabled..."